com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -333,6 +333,7 @@ public static class ShaderKeywordStrings
         internal static readonly string ClusteredRendering = "_CLUSTERED_RENDERING";
         public static readonly string AdditionalLightShadows = "_ADDITIONAL_LIGHT_SHADOWS";
         public static readonly string ReflectionProbeBoxProjection = "_REFLECTION_PROBE_BOX_PROJECTION";
+        public static readonly string ReflectionProbeBoxProjectionOff = "_REFLECTION_PROBE_BOX_PROJECTION_OFF";
```

Comment:
I like `_OFF`

Reply:
Same. It clearly shows what the keyword is about.

Reply:
Seems reasonable, we have both _OFF and _ON. This is also what Unity generates when using attribute [ToggleOff] and [ToggleOn] respectively in shader properties.

https://sourcegraph.com/github.com/Unity-Technologies/Graphics/-/blob/com.unity.render-pipelines.universal/Shaders/ComplexLit.shader#L111

[ToggleOff] generates shader_feature FEATURE_OFF

Reply:
I do like having both `_OFF` and `_ON`, but I think it'll be less breaking to not change the existing keyword. That being said, we should consider doing a pass over the keywords in one big go some time, so that we can make the naming entirely consistent.